ext/storage_sources/s3_store/s3_connection.cpp

```diff
@@ -129,21 +144,16 @@ S3Connection::DeleteObject(const std::string &objectKey) const
     return (1);
```

Comment:
Are we doing a proper error return code as a separate ticket? Is that the part of the return status class object ticket ?

Reply:
Yes, defining the error codes would be part of WT-9059.